recipes/stages/_base_/data/pipelines/semisl_pipeline.py

```diff
@@ -31,3 +33,14 @@
     dict(type="ToTensor", keys=["gt_label"]),
     dict(type="Collect", keys=["img", "gt_label"]),
 ]
+
+train_rand_pipeline = [
```

Comment:
How do you think about separating pipeline file? This file should have only `semisl_pipeline` according to it's filename.

Reply:
We can delete it
Last experiments showed that it is not so useful to use RandAugment

Reply:
But we use other attributes from these file in templates. But templates are not semi supervised :)
![image](https://user-images.githubusercontent.com/68103869/193815313-64e5d18b-93b8-4515-aaf8-8c29661c4818.png)